packages/contracts/contracts/foundry/SphinxPluginTypes.sol

```diff
@@ -228,7 +233,7 @@ contract SphinxPluginTypes {
 
     function sphinxConfigType() external view returns (SphinxConfig memory sphinxConfig) {}
 
-    function leafGasParams() external view returns (SphinxTransaction[] memory txnArray) {}
+    function leafGasParams() external view returns (SphinxTransaction[] memory txnArray, SystemContractInfo[] memory systemContracts) {}
```

Comment:
This is the data structure that describes the input parameters to the `sphinxEstimateMerkleLeafGas` function in `Sphinx.sol`, which estimates the Merkle leaf gas fields. We need to include the system contracts because we can't assume that they've already been deployed on a network when this command is called.